src/psd_tools/api/psd_image.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, data):
         self._record = data
         self._layers = []
         self._tagged_blocks = None
+        self._compatibility_mode = CompatibilityMode.DEFAULT
```

Comment:
Actually, there is `VersionInfo` field in the resource section of a PSD file, that states the file creator like Adobe Photoshop. Perhaps this part should attempt at guessing the default compatibility mode.

```
from psd_tools.constants import Resource
version_info = psd.image_resources.get_data(Resource.VERSION_INFO)
```

Reply:
I can try something like that in bit.